code/datums/supplypacks/contraband.dm

```diff
@@ -46,7 +46,7 @@
 /datum/supply_packs/security/bolt_rifles_militia
  	name = "Surplus militia rifles"
  	contains = list(
- 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle = 3,
+ 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle/mosin = 3, //VOREStation Edit. By request of Ace.
```

Comment:
nooooo just do contraband_vr.dm instead

Reply:
I can't. The gun has been deleted from the game. The path doesn't exist any longer.

Reply:
nevermind this entirely, you saw my Github comments